Tutorials/whatiswot/website/docs/wot/building-blocks.md

```diff
@@ -11,85 +11,99 @@ sidebar_label: Building Blocks
+This is the first tutorial in our Web of Things series. Here, we’ll provide a brief overview of the main building blocks that make up the Web of Things architecture — Thing Description, Binding Templates, Discovery, Scripting API and Profiles. In separate later tutorials, we will explore each individual building block in more detail.
 
-## Concept of Building Blocks
+## What are building blocks?
 
-Let’s first look into the concept of building blocks - they allow the implementation of systems that conform with the abstract WoT Architecture.
+Building blocks allow the implementation of systems that conform with the abstract WoT Architecture. Let us take a closer look at the WoT building blocks.
 
-![wot-building-blocks](/img/12-Building-Blocks/wot-building-blocks.png)
+![wot-building-blocks](/img/12-Building-Blocks/wot-building-blocks-new.png)
 
-Let's take a closer look at the individual WoT building blocks.
+A Thing always has a Thing Description. It is a key building block that provides metadata describing a Thing and its network interface. This TD can be shared directly by the Thing or through the TD Directory — which is what we call Discovery. Things and Consumers can be programmed with the Scripting API.
 
-- A Thing always has a TD.
-- TD is a key building block that provides metadata describing a Thing and its network interface.
-- This TD can be shared directly by the Thing or through a TD Directory.
-- TDs can be restricted by profiles for easier implementation.
-- Things and consumers can be programmed with the scripting API.
-- The WoT Scripting API is optional, and the Binding Templates are informative.
+## Thing Description
 
-## WoT Thing Description
+Let us begin with the Thing Description, or TD — a key component of WoT.
 
-A WoT Thing Description (TD) serves as a crucial aspect of the Web of Things framework.
+All TDs can be processed just like normal JSON documents. Essentially, a TD defines an information model using a semantic vocabulary and a serialized JSON representation. It provides both human- and machine-readable metadata describing the capabilities of a Thing.
 
-All TDs can be represented in JSON-LD and can be processed just like a normal JSON. Essentially, a TD defines an information model using a semantic vocabulary and a serialized JSON representation. It acts as the key building block that provides human and machine-readable metadata describing the capabilities of a Thing and its network interfaces.
-
-The TD describes Thing instances with general metadata such as name, ID, and description. Additionally, TDs can also contain Interaction Affordance metadata and Protocol bindings.
+A TD describes a Thing instance with general metadata such as name, ID, and description. It typically also contains Interaction Affordance metadata and Protocol Bindings (such as Modbus and HTTP), as well as both public and local IP addresses.
```

Comment:
```suggestion
A TD describes a Thing instance with general metadata such as name, ID, and description. It typically also contains Interaction Affordance metadata and Protocol Bindings, such as Modbus or HTTP.
```

IP address point is correct but difficult to explain without explaining it in detail